tests/test_setSpeedTargets.m

```diff
@@ -0,0 +1,27 @@
+function test_suite = test_setSpeedTargets %#ok<*STOUT>
+    try % assignment of 'localfunctions' is necessary in Matlab >= 2016
+        test_functions = localfunctions(); %#ok<*NASGU>
+    catch % no problem; early Matlab versions can use initTestSuite fine
+    end
+    initTestSuite;
+end
+
+function test_setSpeedTargetsBasic()
+
+    run ../initEnv();
+
+    isMT = true;
+    cfg = getTestConfig(isMT);
+
+    speeds = setSpeedTargets(cfg);
+
+    assertEqual(speeds, ones(20, 12) * 28);
+
+    % try when the target are just for the fixation cross
+    cfg.target.type = {'fixation_cross'};
```

Comment:
why fixation targets here?

Reply:
if there are only fixation cross as targets then `setSpeedTargets` must return only zeros.

So it is a second test.

Will break it down into 2 to avoid confusion

Reply:
Ok here I have "updated" the test meaning that now, being WIP, the two tests are checking for the same results (a matrix full of ones).

Added a reminder in #89